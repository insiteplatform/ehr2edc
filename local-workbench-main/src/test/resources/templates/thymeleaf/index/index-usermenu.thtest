%TEMPLATE_MODE HTML5
# ------------------------------------------
%CONTEXT
userMenu = {@eu.ehr4cr.workbench.local.controllers.NavigationItem@about,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@feasibility,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@members,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@myAccount,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@settings,\
			@eu.ehr4cr.workbench.local.controllers.NavigationItem@logout}
# ------------------------------------------
%INPUT (classpath:templates/thymeleaf/index.html)
%FRAGMENT div[@id='user-menu']
# ------------------------------------------
%OUTPUT (classpath:templates/thymeleaf/index/expected/index-usermenu.html)
