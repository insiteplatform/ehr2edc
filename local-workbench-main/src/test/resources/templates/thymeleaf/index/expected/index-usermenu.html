<div id="user-menu" class="action-sheet-container">
    <div position="bottom" class="user-menu text-left action-sheet bottom">
        <ul>
            <li>
                <a href="/testing/app/about">
                    <i class="fa fa-info-circle"></i><span>About InSite</span>
                </a>
            </li>
            <li>
                <a href="/testing/app/feasibility/studies">
                    <i class="fa fa-th-list"></i><span>Query History</span>
                </a>
            </li>
            <li>
                <a href="/testing/app/members/manage">
                    <i class="fa fa-users"></i><span>Members</span>
                </a>
            </li>
            <li>
                <a href="/testing/app/account">
                    <i class="fa fa-user"></i><span>My Account</span>
                </a>
            </li>
            <li>
                <a href="/testing/app/trialDesign/settings">
                    <i class="fa fa-cog"></i><span>Settings</span>
                </a>
            </li>
            <li>
                <a href="/testing/auth/logout">
                    <i class="fa fa-power-off"></i><span>Log out</span>
                </a>
            </li>
        </ul>
    </div>
</div>